CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# Release 2.23.0 13-01-2021
+
+- Automatically configure stream idle timeouts (#457)
```

Comment:
Maybe remove:
- Cleanup usages of `getOrCreateMethod` (#454)
- Move the NOT_FOUND rejection logic to router (#446)
- Remove warning with empty requestTypeName (#448)

Reply:
Removed 1 and 2. Keeping 3 since that was a customer issue.